lighthouse-core/test/index-test.js

```diff
@@ -88,7 +88,7 @@ describe('Module Tests', function() {
   });
 
   it('should return formatted LHR when given no categories', function() {
-    const exampleUrl = 'https://example.com/';
+    const exampleUrl = 'https://www.reddit.com/r/nba';
```

Comment:
lol, any particular reason?

Reply:
I think thats what the devtoolsLog was for those fixtures, which spread to the artifacts, and now to the input URL (though I guess technically only the `finalUrl` assertion would have to change :)